lms/djangoapps/certificates/views/webview.py

```diff
@@ -669,6 +678,29 @@ def render_html_view(request, course_id, certificate=None):  # pylint: disable=t
         return response
 
 
+def _populate_certificate_configuration_from_credentials(configuration, course, mode):
+    """
+    Populates certificate configuration from credentials service.
+
+    If config not found in credentials returns received.
+    """
+    response = get_certificate_configuration_from_credentials(course.id, mode)
+    if response and response.status_code == status.HTTP_200_OK:
+        data = response.json()
+
+        configuration['course_title'] = data.get('title') or configuration['course_title']
+        configuration['is_active'] = data.get('is_active') or configuration['is_active']
```

Comment:
```
if data.get('title'):
        configuration['course_title'] = data['title']
if data.get('is_active'):
        configuration['is_active'] = data['is_active']
```
?

Reply:
I thought it looked better this way. Fixed